tests/test_frame.py

```diff
@@ -2410,3 +2410,15 @@ def test_in_columns() -> None:
     check(assert_type(df.loc[:, cols], pd.DataFrame), pd.DataFrame)
     check(assert_type(df[cols], pd.DataFrame), pd.DataFrame)
     check(assert_type(df.groupby(by=cols).sum(), pd.DataFrame), pd.DataFrame)
+
+
+def test_insert_newvalues() -> None:
+    df = pd.DataFrame({"a": [1, 2]})
+    gh = df.insert(loc=0, column="b", value=None)
+    ab = pd.DataFrame({"col1": [1, 2], "col2": [3, 4]})
+    cd = ab.insert(loc=0, column="newcol", value=[99, 99])
+    ef = pd.DataFrame({"z": [4, 5, 6]})
+    hi = ef.insert(loc=0, column="g", value=4)
+    check(assert_type(gh, None), None)
+    check(assert_type(cd, None), None)
+    check(assert_type(hi, None), None)
```

Comment:
You're going to have to do this using the following style, since `insert()` returns `None`:
```python
assert assert_type(df.insert(loc=0, column="b", value=None), None) is None
```
Our `check` function doesn't accept `None`, and `mypy` looks at functions that return `None` as not returning a value, so you can't do the assignment.

Reply:
done sir